Src/FluentAssertions/Equivalency/Selection/ExcludeMemberByPathSelectionRule.cs

```diff
@@ -23,6 +23,12 @@ public ExcludeMemberByPathSelectionRule(MemberPath pathToExclude)
                 memberToExclude.IsSameAs(new MemberPath(member, parentPath)));
         }
 
+        public void CombinePath(MemberPath nextPath)
```

Comment:
🔧 Usually, a method like `Combine` implies it combines two or more arguments into something new. But I think it just _appends_ something to the existing rule.

Reply:
I renamed it to `AppendPath`. I also renamed to `Combine` method (which I created) on the `MemberPath` to `Join`